app/src/test/java/org/simple/clinic/util/RxErrorsRule.kt

```diff
@@ -0,0 +1,53 @@
+package org.simple.clinic.util
```

Comment:
Do we not need to add this to any of the instrumented tests?

Reply:
Yes, we should. I've added the rule to all instrumentation tests. Please check if the rule chains are setup correctly?

Reply:
Done, and left feedback.